platform/database/factories/OrganizationFactory.php

```
@@ -54,4 +55,21 @@ public function enterprise(): static
             'plan_tier' => PlanTier::Enterprise,
         ]);
     }
+
+    public function withCredits(int $amount = 5): static
+    {
+        return $this->state(fn (array $attributes) => [
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attributes'. (undefined)

(UnusedFormalParameter)